src/Student.java

```diff
@@ -0,0 +1,15 @@
+
+
+public class Student {
+    String nameStudent;
+    String gradeEnter;
+    String matterRecorded;
```

Comment:
Matters should be a list of string, since the requirement says each student can have a minimum of one and maximum of tree matters